specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2279 @@
+{
```

Comment:
There are a bunch of breaking changes reported on this PR. Please fix the changes or get a breaking changes approval and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

Reply:
I have updated the PR, but I am not able to update the labels on it. I also want to add `NewApiVersionRequired` label to this PR

Reply:
@stuti-1807 - Your PR doesn't need a NewApiVersionRequired label because your submission form indicates it is creating a new API version.